test/functional/feature_cltv.py

```diff
@@ -25,7 +25,7 @@
 
 # Reject codes that we might receive in this test
 REJECT_INVALID = 16
-REJECT_OBSOLETE = 17
+# REJECT_OBSOLETE = 17
```

Comment:
When was the first and last use? Might want to remove altogether after investigating that.

Reply:
Last use was removed in your commit fac3e22b18cd29053bc17065fd75db7b84ba6f40 :-)

Reply:
So please remove the constants here as well.